samples/README.md

```diff
@@ -486,6 +488,22 @@ View the [source code](https://github.com/googleapis/python-storage/blob/main/sa
 
 `python storage_enable_versioning.py`
 
+-----
+### FileIO Write-Read
+[![Open in Cloud Shell][shell_img]](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/googleapis/python-storage&page=editor&open_in_editor=samples/snippets/storage_fileio_write_read.py,samples/README.md)
+
+View the [source code](https://github.com/googleapis/python-storage/blob/main/samples/snippets/storage_fileio_write_read.py). To run this sample:
+
+
+`python storage_fileio_write_read.py`
```

Comment:
If the sample needs command-line args to work then I think we should mention that here as well.

Reply:
Yes, done.

Reply:
Per our discussion, we can submit a separate PR to update all command-line args in the readme, if we decide to keep them. 

For now, could you help update the command-line args to `<BUCKET_NAME>` `<BLOB_NAME>` for consistency? Thanks!

i.e. `python storage_fileio_write_read.py <BUCKET_NAME> <BLOB_NAME>`

Reply:
Yes, done.